### ISSUE TITLE
Limit the total value of checks that can be issued each month/week/...

### ISSUE THREAD

**Issue by jonathanvdc** [jonathanvdc/electronic-checkbook#5]
We currently enforce a bound on the total value of an account holder's active unspent checks, but we should also have a bound on the total value of the checks issued to an account holder every month or week or so. We don't want account holder devices to be able to request their entire cap worth of checks, spend them all, then rinse and repeat. That scenario may be convenient for account holders in some situations, but it can be catastrophic if someone steals an account holder device.

So, in summary, we need to make sure that the total values of checks per month/week/... never exceed the account holder device's cap. This only needs to be added to the reference implementation: it has already been specified.
